RockWeb/Blocks/Event/RegistrationEntry.ascx.cs

```diff
@@ -58,6 +58,9 @@ namespace RockWeb.Blocks.Event
     [BooleanField( "Enable Debug", "Display the merge fields that are available for lava ( Success Page ).", false, "", 5 )]
     [BooleanField( "Allow InLine Digital Signature Documents", "Should inline digital documents be allowed? This requires that the registration template is configured to display the document inline", true, "", 6, "SignInline" )]
     [SystemEmailField( "Confirm Account Template", "Confirm Account Email Template", false, Rock.SystemGuid.SystemEmail.SECURITY_CONFIRM_ACCOUNT, "", 7 )]
+	[TextField ( "Family Term", "The term to use for specifying which household or family a person is a member of.", true, "immediate family", "", 8)]
+	[BooleanField( "Force Email Update", "Force the email to be updated on the person's record.", false, "", 9 )]
```

Comment:
I see this new setting is used to control the warning message on screen, but it it actually used somewhere to prevent/force the email on the person's record from being updated?